torch/nn/functional.py

```diff
@@ -446,7 +445,7 @@ def max_pool1d_with_indices(input, kernel_size, stride=None, padding=0,
     See :class:`~torch.nn.MaxPool1d` for details.
     """
     if stride is None:
-        stride = torch.jit.annotate(List[int], [])
+        stride = torch.jit.annotate(List[int], kernel_size)
```

Comment:
The `BroadcastingListx[T]` gets removed and replaced with `List[T]` when the types are resolved during schema-making, so this can just be `stride = kernel_size`